docs/diagrams/NearbyClassDiagram.puml

```diff
@@ -0,0 +1,136 @@
+@startuml
+skinparam backgroundColor #ffffff
+
+class DistrictInfo {
```

Comment:
There might be some missing navigability, like between the NearbyCommand and NearbyCommandUtil class.